apps/judicial-system/backend/src/app/formatters/formatters.spec.ts

```diff
@@ -1398,13 +1404,15 @@ describe('formatDefenderCourtDateLinkEmailNotification', () => {
     const requestSharedWithDefender = RequestSharedWithDefender.COURT_DATE
 
     // Act
-    const res = formatDefenderCourtDateLinkEmailNotification(
+    const res = formatDefenderCourtDateLinkEmailNotification({
       formatMessage,
       overviewUrl,
       court,
       courtCaseNumber,
-      requestSharedWithDefender === RequestSharedWithDefender.COURT_DATE,
-    )
+      requestSharedWithDefender:
+        requestSharedWithDefender === RequestSharedWithDefender.COURT_DATE,
+      defenderSubRole: DefenderSubRole.DEFENDANT_DEFENDER,
```

Comment:
To avoid a lot of redundant Contentful strings and functions, I wanted to test adding a defender sub role to control the string concatenation